netjsonconfig/backends/airos/converters.py

```diff
@@ -0,0 +1,789 @@
+        result = []
+        result.append({
+            'delay': 300,
+            'period': 300,
+            'retry': 3,
+            'status': 'disabled',
+        })
+        return (('pwdog', result),)
+
+
+class Radio(AirOsConverter):
+    netjson_key = 'radios'
+
+    @property
+    def radio(self):
+        return get_copy(self.netjson, self.netjson_key, [])
+
+    @property
+    def wireless(self):
+        return wireless(get_copy(self.netjson, 'interfaces', []))
+
+    def to_intermediate(self):
+        result = []
+        radios = []
+        wireless = {radio(w): w for w in self.wireless}
+        for logic in self.radio:
+            w = wireless.get(logic['name'])
+            if w:
+                user_config = radio_available_mode[mode(w)](logic)
+                radios.append(user_config)
+
+        result.append(radios)
+        result.append(radio_configuration)
+        return (('radio', result),)
+
+
+class Resolv(AirOsConverter):
+    netjson_key = 'dns_servers'
+
+    def host(self):
+        original = get_copy(self.netjson, 'general', {})
+        return {
+            'host': [{
+                'name': original.get('hostname', 'airos'),
+                'status': 'enabled',
+            }],
+        }
+
+    def nameserver(self):
+        result = []
+        original = get_copy(self.netjson, self.netjson_key, [])
+        for nameserver in original:
+            result.append({
+                'ip': nameserver,
+                'status': 'enabled',
+            })
+        return {'nameserver': result}
+
+    def to_intermediate(self):
+        result = []
+        result.append(self.host())
+        result.append(self.nameserver())
+        result.append({'status': 'enabled'})
+        return (('resolv', result),)
+
+
+class Route(AirOsConverter):
+    netjson_key = 'routes'
+
+    def default_routes(self):
+        def is_default_route(interface):
+            try:
+                t = [addr.get('gateway', '') for addr in interface['addresses']]
+                return any(t)
+            except KeyError:
+                return False
+
+        result = []
+        original = [x for x in get_copy(self.netjson, 'interfaces', []) if is_default_route(x)]
+        for interface in original:
+            for address in interface['addresses']:
+                try:
+                    result.append({
+                        'devname': interface['name'],
+                        'gateway': address['gateway'],
+                        'ip': '0.0.0.0',
+                        'netmask': 0,
+                        'status': 'enabled',
+                    })
+                except KeyError:
+                    pass
+        return result
+
+    def to_intermediate(self):
+        result = []
+        routes = []
+        routes = self.default_routes()
+        original = get_copy(self.netjson, self.netjson_key, [])
+        for r in original:
+            network = ip_interface(r['destination'])
```

Comment:
this line is causing the build to fail on python2, see #97 and related fix 61801a12f20358925e04185ef7be9307addf6d39, I suggest you to apply a similar fix